components/eamxx/src/share/io/scream_output_manager.cpp

```diff
@@ -790,13 +825,25 @@ push_to_logger()
   m_atm_logger->info("           Filename prefix: " + m_filename_prefix);
   m_atm_logger->info("                    Run t0: " + m_run_t0.to_string());
   m_atm_logger->info("                   Case t0: " + m_case_t0.to_string());
-  m_atm_logger->info("              Reference t0: " + m_output_control.timestamp_of_last_write.to_string());
+  m_atm_logger->info("              Reference t0: " + m_output_control.last_write_ts.to_string());
   m_atm_logger->info("         Is Restart File ?: " + bool_to_string(m_is_model_restart_output));
   m_atm_logger->info("        Is Restarted Run ?: " + bool_to_string(m_is_restarted_run));
   m_atm_logger->info("            Averaging Type: " + e2str(m_avg_type));
   m_atm_logger->info("          Output Frequency: " + std::to_string(m_output_control.frequency) + " " + m_output_control.frequency_units);
-  m_atm_logger->info("         Max snaps in file: " + std::to_string(m_output_file_specs.max_snapshots_in_file));  // TODO: add "not set" if the value is -1
-  m_atm_logger->info("      Includes Grid Data ?: " + bool_to_string(m_output_file_specs.save_grid_data));
+  switch (m_output_file_specs.storage.type) {
+    case NumSnaps:
+      m_atm_logger->info("             File Capacity: " + std::to_string(m_output_file_specs.storage.max_snapshots_in_file) + " snapshots");  // TODO: add "not set" if the value is -1
```

Comment:
Is there a reason the TODO here isn't added in the PR?

Reply:
I can add that.